db/migrate/20220609221942_remove_orphaned_owners.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOrphanedOwners < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
I think we should still use `up` and `down` and [raise `ActiveRecord::IrreversibleMigration` from `down`](https://guides.rubyonrails.org/active_record_migrations.html#using-the-up-down-methods). Rails can't deduce how to undelete a record.

Reply:
Implemented this